contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,198 @@
+
+Experimentation uses Feature Flags under the hood to handle user allocation to an experiment. You'll notice this when checking if a user is assigned to an experiment.
+
+## Using Experimentation
+
+### Creating an experiment
+
+Guide on how to create an experiment [here](https://docs.google.com/document/d/1DJmSY6ILZFmHhmRLTQ0L2oXGgFZCvGM4DF-zh92DIE8/edit?usp=sharing).
+
+### Running an experiment
+
+TODO: Guide on how to run an experiment here.
+
+Before you can launch an experiment, you need to implement the code for each feature flag variant. On the draft experiment screen, you'll see a snippet telling you how to query for the variant. It looks something like:
+
+```js
+if (posthog.getFeatureFlag('your-chosen-feature-flag-key') === 'control') {
+    // where 'control' is the variant, run your code here
+}
+```
+
+One important gotcha to keep in mind is that feature flag values are loaded asynchronously, so you should use [the onFeatureFlag directive](https://posthog.com/docs/user-guides/feature-flags#onfeatureflags) to check when feature flags are loaded. That is, call the snippet above inside the `onFeatureFlag` callback.
+
+Once you've written your code, it's a good idea to test that each variant behaves as you'd expect. If you find out your implementation had a bug after you've launched the experiment, you lose days of effort as the experiment results can no longer be trusted.
+
+
+
+There's two easy ways to test your changes:
+
+1. Use the PostHog Toolbar
+
+Once you've pushed your experiment code, you can open the PostHog toolbar, click on the feature flags option, and override the feature flag to whichever variant you want, and see that the changes reflect on your screen.
+
+2. Use the web developer console
+
+The experiment draft screen also has a snippet for testing your code. Once you've pushed your experiment code, open the console on your browser, and paste the snippet in. This will override the feature flag for you, and you should be able to see the changes you've made on screen.
+
+Once you're satisfied, you're ready to launch your experiment.
+
+**Note:** The Feature Flag is activated only when you launch the experiment.
+
+While the experiment is running, you can see results on the page. Sometimes, in the beginning of an experiment, results can be skewed to one side. For example, you launched at midnight, so people on the other side of the world are active first, and the first few hours of data you collect is all from them. This can artificially skew results to one variant. While possible, it's important that you don't end your experiments at this stage: the results are probably insignificant. This is known as the peeking problem.
```

Comment:
```suggestion
While the experiment is running, you can see results on the page. Sometimes, in the beginning of an experiment, results can be skewed to one side. For example, if you launched at midnight then people on the other side of the world are active first, and the first few hours of data you collect is all from them. This can artificially skew results to one variant. While possible, it's important that you don't end your experiments at this stage: the results are probably insignificant. This is known as the peeking problem.
```